src/north/CsvToHttp/CsvToHttp.schema.jsx

```diff
@@ -81,6 +81,22 @@ schema.form = {
         <p>
           The unselected keys will be omitted during the mapping.
         </p>
+        <p>
+          <b>Example 1: </b>
+          If the csv file contains 2 headers called header1 and header2, it is possible to specify the name for each field in the
+          http request. It is possible to have in the body of the request a field called header1AfterMapping with the value of the header1
+        </p>
+        <p>
+          <b>Example 2: </b>
+          With the same csv file, it is possible to specify the name for each header in the http request using template string
+          (references to other header as a variable). The syntax is the js one (such as: $
+          {'{value}'}
+          ).It is possible to have in the request&lsquo;s body a field called header1AndHeader2 its value is the concatenation of both
+          header&lsquo;s value.
+          The &quot;Header in csv file&quot; must be filled with the value: $
+          {'{header1} and {header2}'}
```

Comment:
I think that the examples here are a little bit complicated for a user who starts using this connector. Can you try to improve it please ?